packages/default/scss/notification/_layout.scss

```diff
@@ -72,31 +72,27 @@
     }
 
     // RTL
-    &[dir='rtl'] .k-notification-wrap,
-    .k-rtl .k-notification-wrap {
-        padding-left: $notification-padding-end;
-        padding-right: $notification-padding-x;
+    .k-notification,
+    .k-notification-container {
+        &[dir='rtl'],
+        .k-rtl & {
```

Comment:
I think these selectors can be simplified a bit, since the `.k-rtl` class is always applied to an enclosing element (typically the `<html>` / `<body>` tags). Thus instead of generating 4 rules for each nested selector, the enclosing styles could be

```scss
.k-notification-container[dir='rtl'],
.k-rtl .k-notification {
   ...
}
```

This should work because .k-rtl is used only in jQuery widgets that don't have a notification container, otherwise the dir=rtl is set to only the k-notification-container class.